docs/guides/resource-management.rst

```diff
@@ -63,11 +64,12 @@ Setting Workflow Resources
 --------------------------
 
 Resources can also be configured at the workflow definition level using the same syntax as with tasks, with one difference - the ``sdk.Resources()`` object my additionally specify a number of nodes to be requested for the workflow. The full parameter list is therefore:
+
 * ``cpu``: number of cores.
 * ``memory``: amount of RAM (bytes).
 * ``disk``: disk space (bytes).
 * ``gpu``: whether access to a gpu unit is required (``1`` if a GPU is required, ``0`` otherwise).
-* ``nodes``: the number of nodes requested.
+* ``nodes``: the number of nodes requested (needs to be greater than ``1`` when custom images are used and specifies the maximum number of nodes per different image type in that case).
```

Comment:
I know we have an ongoing discussion with regards to resources at `workflow` decorator, but leaving it aside, I suggest some rephrasing of this line, in case my understand is accurate. Also, I am assuming that if omitted, it is assumed `1` 

> `nodes`: indicates the maximum number of nodes that may be allocated throughout the execution of this worklow - Must be a positive integer and must be greater than `1` node when using custom images in `sdk.task` resources. If omitted, it defaults to ...